db/migrate/20160109015425_add_password_digest_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPasswordDigestToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :password_digest, :string
```

Comment:
when doing a pull request. its a bit nicer to combine together all the migrations that apply to the same model